kong/clustering/compat/checkers.lua

```diff
@@ -37,6 +37,18 @@ local compatible_checkers = {
               dp_version, log_suffix)
           end
         end
+
+        if plugin.name == 'prometheus' then
```

Comment:
this file is meant for complex transformations or deprecation message, for this fix we can just add it here https://github.com/Kong/kong/blob/master/kong/clustering/compat/removed_fields.lua#L147

Reply:
@fffonion  noted. I just made the change

Reply:
No worries :) 
I have Just added it back